plugins/quay/src/components/utils.ts

```diff
@@ -0,0 +1,42 @@
+import { VulnerabilitySeverity } from '../types';
+
+const byteUnits = ['B', 'kB', 'MB', 'GB', 'TB'];
+
+export const SEVERITY_COLORS = new Proxy(
+  new Map([
+    [VulnerabilitySeverity.Critical, '#7D1007'],
+    [VulnerabilitySeverity.High, '#C9190B'],
+    [VulnerabilitySeverity.Medium, '#EC7A08'],
+    [VulnerabilitySeverity.Low, '#F0AB00'],
+    [VulnerabilitySeverity.None, '#3E8635'],
+  ]) as any,
+  {
+    get: (o: Map<VulnerabilitySeverity, string>, k: VulnerabilitySeverity) =>
+      o.has(k) ? o.get(k) : '#8A8D90',
+  },
+);
+
+export function formatDate(date: string | number): string {
```

Comment:
It's a bit of a nit-pick but you can rewrite this function to follow the single-responsibility principle like this:
```ts
export function formatDate(date: string | number): string {
  if (isNotAvailable(date)) {
    return 'N/A';
  }

  const adjustedDate = adjustDate(date);
  return formatAdjustedDate(adjustedDate);
}

function isNotAvailable(date: string | number): boolean {
  return date === -1;
}

function adjustDate(date: string | number): number | string {
  return typeof date === 'number' ? date * 1000 : date;
}

function formatAdjustedDate(adjustedDate: number | string): string {
  return new Date(adjustedDate).toLocaleString('en-US', {
    timeZone: getUserTimeZone(),
    timeStyle: 'short',
    dateStyle: 'medium',
  });
}

function getUserTimeZone(): string {
  return new Intl.DateTimeFormat().resolvedOptions().timeZone;
}
```

Reply:
I've just extracted the `getUserTimeZone` function since I see how the implementation could change in the future.

Reply:
Perfect!